src/oscar/templates/oscar/catalogue/reviews/review_form.html

```diff
@@ -16,7 +16,7 @@
                     </label>
                     <div class="controls">
                         {{ form.score }}
-                        <div class="star-rating {% if form.score.value == '1' %}One{% elif form.score.value == '2' %}Two{% elif form.score.value == '3' %}Three{% elif form.score.value == '4' %}Four{% elif form.score.value == '5' %}Five{% endif %}">
+                        <div class="star-rating {{ form.score.value|as_star }}">
```

Comment:
I think this [should be](https://github.com/django-oscar/django-oscar/blob/master/src/oscar/templatetags/reviews_tags.py#L7) `as_stars`?

Reply:
Indeed 🙈 Thanks for spotting this!